setup.py

```diff
@@ -0,0 +1,70 @@
+import re
+
+from setuptools import find_namespace_packages, setup
+
+
+# Ensure we match the version set in optimum/executorch/version.py
+filepath = "optimum/executorch/version.py"
+try:
+    with open(filepath) as version_file:
+        (__version__,) = re.findall('__version__ = "(.*)"', version_file.read())
+except Exception as error:
+    assert False, "Error: Could not open '%s' due %s\n" % (filepath, error)
+
+INSTALL_REQUIRE = [
+    # "optimum~=1.24",
```

Comment:
Why is it commented?

Reply:
needs to install optimum from source, we can upgrade to optimum v1.24 once released